.codecarbon.config

```diff
@@ -0,0 +1,4 @@
+[codecarbon]
+log_level = ERROR
+tracking_mode = process
+save_to_file = False
```

Comment:
I'm not sure this will work for people installing it (it looks like they support having it in either the home or working directory...). That said I'm not sure there's an easy fix so we should just merge for now and I'll raise an issue to flag it.